plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftClient.java

```diff
@@ -304,9 +309,30 @@ public Connection getConnection(ConnectorSession session, JdbcSplit split, JdbcT
 
     @Override
     public Optional<String> getTableComment(ResultSet resultSet)
+            throws SQLException
     {
-        // Don't return a comment until the connector supports creating tables with comment
-        return Optional.empty();
+        return Optional.ofNullable(emptyToNull(resultSet.getString("REMARKS")));
```

Comment:
Does Redshift expect an empty string for NULL comment? I'm asking this question because it's different from PostgreSQL.

Reply:
fixed. it returns NULL for empty comment